packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -11,872 +11,13 @@ import 'package:flutter/rendering.dart';
 import 'automatic_keep_alive.dart';
 import 'basic.dart';
 import 'framework.dart';
-import 'selection_container.dart';
-
-export 'package:flutter/rendering.dart' show
-  SliverGridDelegate,
-  SliverGridDelegateWithFixedCrossAxisCount,
-  SliverGridDelegateWithMaxCrossAxisExtent;
+import 'scroll_delegate.dart';
 
 // Examples can assume:
```

Comment:
Are there still examples in this file that need all of these (I noticed most of these were duplicated over in scroll_delegate.dart, wondering if some should be removed from here..)

Reply:
Oh right I meant to check that. Thank you!